conda-store-server/conda_store_server/server/app.py

```diff
@@ -198,9 +199,9 @@ def trim_slash(url):
         app = FastAPI(
             title="conda-store",
             version=__version__,
-            openapi_url=os.path.join(self.url_prefix, "openapi.json"),
-            docs_url=os.path.join(self.url_prefix, "docs"),
-            redoc_url=os.path.join(self.url_prefix, "redoc"),
+            openapi_url=posixpath.join(self.url_prefix, "openapi.json"),
```

Comment:
I'll admit using `os.path...` for urls is a hack and I should have used something like yarl for example. Thoughts? Since url paths are not the same as file paths? Maybe posixpath is the right thing to use here. It certainly seems better than os.path.

Reply:
A separate entity for URLs would be better, yes. But I'm not going to block this PR just because of this issue, since everything seems to work fine.